examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -224,6 +234,24 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
         }
       };
     }
+    
+    function handleByTimeRequest(parameters, categoryName) {
```

Comment:
Can you put a reason as a comment why the following check is needed?

Reply:
@ilvry you only call this `handleByTimeRequest` function in `heatmap` and `pinmap` cases, right? Then no need to check that again, I think.